app/common/templates/common/macros/status.html

```diff
@@ -42,3 +42,29 @@
   %}
   {{ govukTag({"text": status, "classes": classMap[status] }) }}
 {% endmacro %}
+
+{% macro submissionStatusTag(status, overdue=False) %}
+  {%
+    set classMap = {
+        enum.submission_status.NOT_STARTED: "govuk-tag--grey" if not overdue else "govuk-tag--yellow",
+        enum.submission_status.IN_PROGRESS: "govuk-tag--light-blue" if not overdue else "govuk-tag--yellow",
+        enum.submission_status.COMPLETED: "",
+        enum.submission_status.READY_TO_SUBMIT: "govuk-tag--blue" if not overdue else "govuk-tag--yellow",
+        enum.submission_status.AWAITING_SIGN_OFF: "govuk-tag--purple",
+    }
+  %}
+  {{ govukTag({"text": status if not overdue else "Overdue", "classes": classMap[status] }) }}
+{% endmacro %}
```

Comment:
This split between codified statuses and the dynamic overdue status feels a bit like it might leak and be hard to keep track of 🤔